### PR TITLE
Milestone 7: Add mobile nav-menu

### PR DESCRIPTION
**Pull request by yonas44**
- Setup JavaScript linters.
-  Add mobile navigation.
- Used JavaScript to manipulate DOM.

Thankyou for reviewing my PR!

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -9,12 +9,23 @@
...
+  background: white;
   display: flex;
-  padding: 10px 35px 0 24px;
+  z-index: 4;
+  padding: 20px 20px;
+  width: 90vw;
   align-items: center;
   justify-content: space-between;
+  position: fixed;
 }
```
Comment (ShahierNashaat) [inline style.css:28]: - [x] Kindly give the `mobile-nav` `top: 0;` so it sticks at the top of the page, and not a little bit down at the top of the page.

File: style.css [hunk 2]
```diff
@@ -55,28 +66,59 @@
...
 
+.head .open {
+  display: block;
+  background: #e5e5e5;
+  height: 755px;
+  border-bottom-left-radius: 35px;
+  border-bottom-right-radius: 35px;
+  z-index: 5;
+}
```
Comment (ShahierNashaat) [inline style.css]: - [x] Kindly give the mobile menu `height: 100vh` so it takes the whole height of the page and looks like the Figma design.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @yonas44, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Good overall design ✔️
- Your code is clean and well documented ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (yonas44) [reply style.css]: Okay, thanks for the feedback @ShahierNashaat!

**Review by mikethreels — APPROVED** [review]
Hi Metron, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- good job following the instructions of the previous CR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by yonas44** [comment]
Thank you @mikethreels I will proceed with the merge.

**Pull request merged**